app/views/govuk_publishing_components/components/_search.html.erb

```diff
@@ -1,6 +1,9 @@
 <%
   class_name = "gem-c-search--on-white"
-  class_name = "gem-c-search--on-govuk-blue" if local_assigns.include?(:on_govuk_blue)
+  class_name = "gem-c-search--on-govuk-blue" if local_assigns.include?(:on_govuk_blue) && 
```

Comment:
I can I suggest we do the following, for consistency
```suggestion
class_name = "gem-c-search--on-govuk-blue" if local_assigns[:on_govuk_blue].eql?(true)
```

Reply:
that was quick :)

Reply:
much more elegant!